README.md

```diff
@@ -24,7 +31,7 @@ You have to write `[...[]].reverse().find()`.
 
 As a result, there is a third issue:
 
-3. **unnecessary spread**
+3. **unnecessary copy**

```

Comment:
.reverse mutates, so it’s not making a copy, it’s changing the original:
```suggestion
3. **unnecessary mutation**
```

The copy is done by the spread, but the problem isn’t the copy by itself, it’s that you have to copy to avoid the mutation.

Reply:
Yep. The paragraph is a bit confusing. I'll rewrite them to be more clear.